src/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -53,25 +53,33 @@ public object CreateFake(Type typeOfFake, Action<IFakeOptions> optionsBuilder)
         /// </returns>
         public IList<T> CollectionOfFake<T>(int numberOfFakes)
         {
+            return this.CollectionOfFake<T>(numberOfFakes, x => { });
+        }
+
+        public IList<T> CollectionOfFake<T>(int numberOfFakes, Action<IFakeOptions<T>> optionsBuilder)
+        {
             var result = new List<T>();
 
             for (var i = 0; i < numberOfFakes; i++)
             {
-                result.Add(this.CreateFake<T>(x => { }));
+                result.Add(this.CreateFake(optionsBuilder));
             }
 
             return result;
         }
 
-        public IList<object> CollectionOfFake(Type typeOfFake, int numberOfFakes)
+        public IList<object> CollectionOfFake(Type typeOfFakes, int numberOfFakes)

```

Comment:
I see that the existing parameter in `IFakeCreatorFacade` is plural so there is already inconsistency. I think the cleanest way is to remove this change from this commit, since it's unrelated to adding the new overloads, and add another commit which makes everything consistent, whether singular or plural. I have no preference which.